spyder/config/base.py

```diff
@@ -68,6 +68,11 @@ def running_in_ci():
     return bool(os.environ.get('CI'))
 
 
+def running_installer_test():

```

Comment:
I think this function could also go in `utils/installers.py` because it's only used there.